src/app/client/package.json

```diff
@@ -48,7 +48,7 @@
     "@angular/compiler-cli": "14.3.0",
     "@angular/core": "14.3.0",
     "@angular/forms": "14.3.0",
-    "@angular/localize": "14.3.0",
+    "@angular/localize": "16.0.6",
```

Comment:
@sedin-tushar please don't change any "@angular" related upgrade

Reply:
The @angular versions are downgraded to previous versions